app/lib/matador/components/dot.tsx

```diff
@@ -0,0 +1,17 @@
+interface DotProps {
```

Comment:
You moved all the props in the same file, doing great.

Just a last consideration on this, you should export components props like the ones you just refactored.

Reply:
Fixed with 35c6421357701641e9eedeec5819dece439e234c